source/tutorial.rst

```diff
@@ -0,0 +1,111 @@
+Language Tour
+=====================================
+
+ManimDSL is a small, statically typed language designed to leverage the Manim visualisation library to enable educators to succinctly express and visualise algorithms. The core language is small by design.
+
+This tour of the language will take you through some of the basic constructs of the language, many of which will be familiar to those with any programming background.
+It is assumed you have at least very basic knowledge of programming in any other language (such as Python, JavaScript or Java). 
+
+
+Note that this is an open source project and we welcome suggestions and improvements! If you would like to make a contribution feel free to submit a pull request `here <https://github.com/ManimDSL/ManimDSLCompiler/tree/master/>`_.
+
+
+The Basics
+----------
+
+Variables
+~~~~~~~~~~~~~~~~~~~~~
+
+Declaring and assigning a variable looks a lot like it does in any other language. 
+
+Note that depending on your code style during declaration you may choose to omit or specify the type of the variable.
+
+.. code :: javascript
+    
+    let x = 4;
+    let y : number = 4;
+
+
+Arithmetic
+~~~~~~~~~~~~~~~~~~~~~
+
+Arithmetic expressions also look a lot like in any other programming language (for good reason!). 
+
+.. code:: javascript
+    
+    let x = 4.5;
+    x = (x * 2) + 3; // x = 12
+  
+    
+Constructors
+~~~~~~~~~~~~~~~~~~~~~
+
+Data structures baked into the language have constructors. These can be invoked using the ``new`` keyword.
+
+Note that if a data structure (as below) takes generic type arguments in their constructor they must not be omitted.
+
+.. code:: javascript
+    
+    let stack = new Stack<number>;
+
+
+Control structures
```

Comment:
Could we comment out these two sections for now? It's being deployed and I'm not sure it'll make much sense to the user. Alternatively, maybe replace 'TODO' with 'Work in progress!'